build/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.4'
+services:
+  postgres:
+    image: postgres
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_PASSWORD=admin
+    volumes:
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+  account:
+    build:
+      context: ../
+      dockerfile: ./build/Dockerfile
+      target: account
+    depends_on:
+      postgres:
+        condition: service_healthy
+  server:
+    build:
+      context: ../
+      dockerfile: ./build/Dockerfile
+      target: server
+    ports:
+      - 11000:11000
+      - 11001:11001
+      - 12000:12000
+      - 12001:12001
+      - 13000:13000
```

Comment:
This actually only needs to be `13000`, I literally changed this last night to unify the gRPC server.

Reply:
Sorry - didn't quite get this. Do you mean that only 13000 should be open now?

In console (updated from master) I see:
server_1    | time="2021-01-29 20:10:24" level=info msg="SHIPGATE waiting for requests on 0.0.0.0:13000\n"
server_1    | time="2021-01-29 20:10:24" level=info msg="PATCH waiting for connections on 0.0.0.0:11000\n"
server_1    | time="2021-01-29 20:10:24" level=info msg="DATA waiting for connections on 0.0.0.0:11001\n"
server_1    | time="2021-01-29 20:10:24" level=info msg="LOGIN waiting for connections on 0.0.0.0:12000\n"
server_1    | time="2021-01-29 20:10:24" level=info msg="CHARACTER waiting for connections on 0.0.0.0:12001\n"

So I thought that client still connects to patch/character/etc servers. Or am I missing something?

Reply:
Exactly. The shipgate server used to run two separate gRPC servers - one on 13000 and the other on 13001. I made a commit yesterday that combined them so that the gRPC service listening on port 13000 handles both services. So port 13001 no longer needs to be mapped

Reply:
Fixed